src/Core/Entity/Image/InputImage.php

```diff
@@ -19,6 +20,9 @@ class InputImage
     /** @var string */
     protected $sourceImageMimeType;
 
+    /** @var array Associative array that holds basic image info. Consider in the future create an imageInfo class. */
+    protected $imageInfo;
```

Comment:
Yes definitely, we need to move this property into a sperate class, call it Core\Entity\Image\ImageMetaInfo.php 
and move the method related to this new class.

Reply:
Awesome, will do